dynamiqs/sesolve/sesolve.py

```diff
@@ -163,8 +163,8 @@ def sesolve(
 
     # get saved tensors and restore correct batching
     if result.ysave is not None:
-        result.ysave = result.ysave.squeeze(0, 1)
+        result.ysave = result.ysave.squeeze(0, 1, 2)
     if result.exp_save is not None:
-        result.exp_save = result.exp_save.squeeze(0, 1)
+        result.exp_save = result.exp_save.squeeze(0, 1, 2)
```

Comment:
Ho because you also batched `H` along an extra dimension for `sesolve`. What about making moving small batching functions specifc to `sesolve` on top of the file? Otherwise we need to fix all the comments of the SE solvers 😭